engine/samples/imgui/main.cpp

```diff
@@ -30,6 +110,14 @@ int main()
     cubos.system(exampleWindowSystem).tagged("cubos.imgui");
     /// [Adding the system]
 
+    /// [Filling the dummy resource 2]
+    cubos.addResource<DummyResource>();
```

Comment:
You could initialize the resource here btw, and thus remove the need for a startup system:
```suggestion
    cubos.addResource<DummyResource>(DummyResource{
        .integer = 1337,
        /*...*/
    });
```